percy/__init__.py

```diff
@@ -19,9 +19,9 @@ def percy_screenshot(driver, name, **kw):
     if driver.__class__.__name__ != "WebDriver":
         raise UnsupportedWebDriverException("Provided driver is not supported")
 
-    if driver.command_executor.__class__.__name__ == "RemoteConnection":
+    if "RemoteConnection" in driver.command_executor.__class__.__name__:
```

Comment:
any spec that needs to be updated as part of it?

Reply:
Added specs